content/en/docs/getting-started/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Getting Started
+description: >-
+  Depending on your role, you might be looking for a different way to get 
+  started with OpenTelemetry. Find your way on how to get started with
+  OpenTelemetry.
+spelling: cSpell:ignore otel
+aliases: [/about, /docs/concepts/overview, /otel]
+weight: 1
+---
+
+Before you deep dive into your role specific _Getting Started_, you can try out
+our official [OpenTelemetry Demo](/docs) to **see** how observability with
```

Comment:
Should this be here? https://github.com/open-telemetry/opentelemetry-demo

Reply:
YEs, or we should have a /demo page somewhere at the website?

Reply:
A deployed demo would be pretty cool. Will ask in slack